db/migration/1688372371221-AddPresentationToVariables.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPresentationToVariables1688372371221
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE variables
+                ADD COLUMN schemaVersion INT NOT NULL DEFAULT 1,
+                ADD COLUMN processingLevel ENUM('minor', 'major'),
```

Comment:
Enum fields are thought to be bad news in MySQL since they're quite annoying to change later and can have surprising properties.

Better to use a short `VARCHAR(30)` here as well.

If you want to, you can add a check constraint on it, they're new to MySQL since 8.0.16 (we're on 8.0.28):

```sql
ALTER TABLE variables
ADD CONSTRAINT checkProcessingLevel CHECK (processingLevel IN ('minor', 'major');
```

Reply:
Good to know, changed it to `VARCHAR(30)`. (I didn't add the constraint, we validate this field in ETL anyway)